oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -180,11 +183,58 @@ def _create_relateditem_audio(self, mi: MediaFile) -> MODSv34:
 
         return ri
 
+    def _populate_narrator_image(self):
+        for img in MediaFile.objects.filter(
+            Q(item=self._item) & Q(file_type__file_code="image_submaster")
```

Comment:
This probably could be simplified to not use Q:
filter(item=self._item, file_type__file_code="image_submaster)`
Assuming output is unchanged and tests pass :)